src/Confluent.Kafka/Consumer.cs

```diff
@@ -316,13 +319,13 @@ public void Poll()
         public event EventHandler<Message<TKey, TValue>> OnMessage;
 
         /// <summary>
-        ///     Returns the current partition assignment as set by Assign.
+        ///     The current partition assignment as set by Assign.
```

Comment:
Nitpick but I think it is clearer with the Returns the  prefix.
Alternatively just use the returns tag